openapi/aro_hcp/v1alpha1/openapi.json

```diff
@@ -1956,6 +1960,28 @@
           }
         }
       },
+      "AzureNodePoolOsDisk": {
+        "description": "AzureNodePoolOsDisk defines the configuration of a Node Pool's OS disk.",
+        "properties": {
+          "persistence": {
+            "description": "Specifies the type of OS disk for the nodes in the Node Pool.\nValid values ared:\n* Persistent\n* Ephemeral\nIf not specified, Persistent OS Disks are used.",
+            "type": "string"
+          },
+          "size_gibibytes": {
+            "description": "The size in GiB to assign to the OS disks of the\nNodes in the Node Pool. The property\nis the number of bytes x 1024^3.\nIf not specified, OS disk size is 30 GiB.",
```

Comment:
The default is actually now 64 GiB. We should change to reflect the real value.

If you've seen the outdated 32GiB value in the CS repository please open a MR to update it there too.